src/session.rs

```diff
@@ -896,6 +899,17 @@ impl Session {
 
         true
     }
+
+    pub fn is_request_keyframe_possible(&self, kind: KeyframeRequestKind) -> bool {
+        // TODO: It's possible to have different set of feedback enabled for different
+        // payload types. I.e. we could have FIR enabled for H264, but not for VP8.
+        // We might want to make this check more fine grained by testing which PT is
+        // in "active use" right now.
```

Comment:
Agreed with this `TODO` it should be based on the media so the signature here would change to 

```rust
pub fn is_request_keyframe_possible(&self, kind: KeyframeRequestKind, pt: Pt) -> bool
```

Reply:
Yeah, but that makes for a public API change I don't want. This is what I mean with "active use". We need to somewhere record which PT is in current use. Hence I left it as TODO for now.

Reply:
Hmm, does this even make sense `is_request_keyframe_possible` is a `StreamRx` question, does it even make sense to ask at the session level?

Reply:
It's for `media/Writer`. The high level API.

Reply:
But then wouldn't you ask as a specific media/Writer about it?

Reply:
Not really necessary. The remote side is sending on _some PT_ and the high level API doesn't care what it is. It just want a keyframe. Rid is the only thing we need to provide.

Reply:
I don't think that makes much sense, but that hasn't changed in this PR so no point in debating that here